### ISSUE TITLE
Metadata file is not correct?

### ISSUE THREAD

**Issue by JulienPeloton** [maxmahlke/rocks#36]
I have metadata mismatch with online resources. I started with a fresh installation of rocks (version 1.9.7, cache folder removed):

```python
import rocks

data = rocks.Rock('Mahlke', skip_id_check=False)

data.ssocard.version
# '1.1.10'

data.parameters.dynamical.orbital_elements.semi_major_axis.value
# 3.00691

data.parameters.dynamical.orbital_elements.semi_major_axis.unit
# km
```

The unit is wrong clearly. ssodnet recently made some change in the units, but if I look online to the most recent version (https://ssp.imcce.fr/data/metadata_asteroid.json) which is also `1.10.0` at the time of writing, the unit is correctly stated in `au`.

I do not understand then why `rocks` gives a different answer given that the metadata must be taken from the same file? https://github.com/maxmahlke/rocks/blob/db5656ce83058b52df84a4cdd5fff37df91bf8a2/rocks/metadata.py#L57

**Comment by JulienPeloton** [comment]
ahh got it -- the filename has changed! before it was https://ssp.imcce.fr/data/metadata_asteroid.json (and this is what rocks assumes), and now it is https://ssp.imcce.fr/data/metadata_asteroid.json (this is what rocks should use)! I will open a PR.

**Comment by maxmahlke** [comment]
Fixed in version `1.9.8`.

**Comment by JulienPeloton** [comment]
Thank you :-)
